filter-nm/lang/zh_CN.ts

```diff
@@ -36,20 +36,21 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Natural movement filter by Tom Brazier: Cancels higher frequency noise and the natural tendency for our heads to drift even when we think we are sitting still.</source>
-        <translation type="unfinished"></translation>
+        <source>Natural movement filter by Tom Brazier: 过滤高频噪声与我们头部的自然飘移, 即便我们认为我们保持不动.</source>
```

Comment:
need change

Reply:
You translated the string `<source> ... </source>`
But you need to translate the string `<translation> ... </translation>` instead

Reply:
It has been fixed.